blackjack.js

```diff
@@ -31,7 +88,22 @@ const calcPoints = (hand) => {
  */
 const dealerShouldDraw = (dealerHand) => {
```

Comment:
At the moment it will not work because of the changes needed to `calcPoints`, but it looks like it will work